Sources/Mustache/Template+Parser.swift

```diff
@@ -306,16 +306,44 @@ extension MustacheTemplate {
         var nameParser = Parser(String(text))
         let string = nameParser.read(while: self.sectionNameCharsWithoutBrackets)
         if nameParser.reachedEnd() {
-            return (text, nil)
+            return (text, [])
         } else {
+            var transforms: [Substring] = [string]
+            var parameterName: Substring?
+
+            func parseTransforms() throws {
+                if let previous = parameterName {
+                    transforms.append(previous)
+                }
+                parameterName = nameParser.read(while: self.sectionNameCharsWithoutBrackets)
+                switch nameParser.current() {
+                case ")":
+                    // Transforms are ending
+                    nameParser.unsafeAdvance()
+                    // We need to have a `)` for each transform that we've parsed
+                    var endCount = 1
+                    while endCount < transforms.count {
```

Comment:
You could probably use `nameParser.read(while: ")")` here. It returns the number of times you read a character. Although I've just noticed the comment header for that function is wrong